circuit_maintenance_parser/output.py

```diff
@@ -139,12 +154,18 @@ class Maintenance(BaseModel, extra=Extra.forbid):
     end: StrictInt
     stamp: StrictInt
     organizer: StrictStr
+    _metadata: Metadata = PrivateAttr()
 
     # Non mandatory attributes
     uid: StrictStr = "0"
     sequence: StrictInt = 1
     summary: StrictStr = ""
 
+    def __init__(self, **data):
+        """Initialize the Maintenance object."""
+        self._metadata = data.pop("_metadata")
```

Comment:
This throws KeyError if _metadata key isn't present, thus this change becomes a breaking change. 

Some of my existing tests used in my projects start to throw KeyError exception. For example:

```
self = Maintenance()
data = {'account': '12623', 'circuits': [{'circuit_id': '297487', 'impact': 'NO-IMPACT'}], 'end': 1649073600, 'maintenance_id': 'GIN-CHG0042605', ...}

    def __init__(self, **data):
        """Initialize the Maintenance object."""
>       self._metadata = data.pop("_metadata")
E       KeyError: '_metadata'

../../dev/MNA/lib/python3.10/site-packages/circuit_maintenance_parser/output.py:166: KeyError
```

Reply:
It seems like Maintenance dataclass is now expecting _metadata to be supplied by users. can't it default to None?

Reply:
thanks @csessh for reporting. I'm going to check and fix.

Reply:
checking the code, I see that the `_metadata` is injected by the `Processors` automatically, so I don't get why don't you get it in your tests.